tests/spin/test_spin_hamiltonian.py

```diff
@@ -782,3 +783,826 @@ def test_fermi_hubbard_hamiltonian_matrix(shape, n_cells, t, coulomb, expected_h
     fermi_hub_ham = fermi_hubbard(lattice=shape, n_cells=n_cells, hopping=t, coulomb=coulomb)
 
     qml.assert_equal(fermi_hub_ham, expected_ham)
+
+
+def test_interaction_parameter_error_emery():
+    r"""Test that an error is raised when the provided interaction parameters are wrong of wrong shape
+    emery Hamiltonian."""
+    n_cells = [4, 4]
+    lattice = "Square"
+    with pytest.raises(
+        ValueError,
+        match=re.escape(
```

Comment:
What's `re.ecsape()`? Never seen it before 😅

Reply:
https://docs.python.org/3/library/re.html#re.escape